openpype/hosts/nuke/api/lib.py

```diff
@@ -2549,6 +2563,31 @@ def sync_module(cls):
         return cls._sync_module
 
 
+def duplicate_node(node):
+    reset_selection()
+
+    # select required node for duplication
+    node_orig = nuke.toNode(node.name())
+    node_orig.setSelected(True)
+
+    # copy selected to clipboard
+    nuke.nodeCopy("%clipboard%")
+
+    # reset selection
+    reset_selection()
+
+    # paste node and selection is on it only
+    nuke.nodePaste("%clipboard%")
+
+    # assign to variable
+    dupli_node = nuke.selectedNode()
```

Comment:
Would this the the same?
```suggestion
    # paste node and receive it
    dupli_node = nuke.nodePaste("%clipboard%")
```

Reply:
Yeah it seems like logic way to approach it, but it is actually  breaking the process.

Reply:
I tried several approaches with clipboard, and all fail for me. What about just reconnecting the Slate node and connecting it back after the publish? If publish fails on the slate generation, this would modify the script - not great. But swapping input + output should work.

Reply:
Just to chime in here, I had th esame problem when dealing with the slate workflow here. For some reason the slate node does not get selected after copypaste. tested in 13.1 and 12.0. I got around it connecting and restoring previous connection to the already present slate. Dunno what's breaking stuff. The weird behaviour started to happen when OP switched from 3.8 to 3.9 though

Reply:
also, scripting in editor the same thing just works. It's only a problem if it's OP doing the copypaste